smartsim/_core/launcher/step/step.py

```diff
@@ -107,3 +116,49 @@ def add_to_batch(self, step: Step) -> None:
         :type step: Step
         """
         raise SmartSimError("add_to_batch not implemented for this step type")
+
+
+_StepT = t.TypeVar("_StepT", bound=Step)
+
+
+def proxyable_launch_cmd(
+    fn: t.Callable[[_StepT], t.List[str]], /
+) -> t.Callable[[_StepT], t.List[str]]:
+    @functools.wraps(fn)
+    def _get_launch_cmd(self: _StepT) -> t.List[str]:
+        original_cmd_list = fn(self)
+
+        if not CONFIG.telemetry_enabled:
+            return original_cmd_list
+
+        if self.managed:
+            raise UnproxyableStepError(
```

Comment:
Hmmmm.... I thought I wrote a test to test explicitly this....let me look into why this wasn't included in the coverage

Reply:
Confirmed that I do have a `test_proxy_launch_cmd_decorator_errors_if_attempt_to_proxy_a_managed_step` test in the `test_telemetry_monitor.py` test module!

I think because the new test modules were added after the CI refactor that split our tests into groups, none of the runners are picking up the new test modules and thus they are not contributing to coverage. Need to look into this a bit further, but that is my running hypothesis!

Reply:
Great